.github/workflows/test-swaps.yml

```diff
@@ -0,0 +1,140 @@
+name: Test sending swaps transactions
```

Comment:
The naming throughout the PR here is using the term "swaps". This shouldn't be a term that we use imo because it isn't descriptive of what's being tested. We're testing AllowSpend and SpendTransaction in these tests. Those txn types can be used for swaps and lots of other use cases (delegated spending, metagraph txns from metagraph address, etc.).

Reply:
Sure, I'll rename it